mln/admin/__init__.py

```diff
@@ -99,10 +105,10 @@ def get_settings_inlines(obj):
 friend_cond_admin.list_display_links = "networker",
 friend_cond_admin.search_fields = "networker", "condition", "success_body__subject", "success_body__text", "failure_body__subject", "failure_body__text", "source__source"
 
-trigger_admin = make_inline(NetworkerMessageTrigger, NetworkerMessageAttachment)
-trigger_admin.list_display = "networker", "body", "trigger", "source", "notes"
-trigger_admin.list_display_links = "body",
-trigger_admin.search_fields = "networker", "body__subject", "body__text", "trigger", "source", "notes"
+reply_trigger_admin = make_inline(NetworkerReplyTrigger, MessageTemplateAttachment)
+reply_trigger_admin.list_display = "networker", "body", "message_body", "message_attachment", "trigger"
+reply_trigger_admin.list_display_links = "body",
+reply_trigger_admin.search_fields = "networker", "body__subject", "body__text", "trigger", "source", "notes"

```

Comment:
With the old `NetworkerMessageTrigger` being split up into `MessageTemplate` and `NetworkerReplyTrigger`, this should be `make_inline(MessageTemplate, MessageTemplateAttachment)`. If you go the suggested way of making the triggers inline to `MessageTemplate`, it would become ``make_inline(MessageTemplate, MessageTemplateAttachment, NetworkerReplyTrigger)`.

Reply:
You could however keep this as an additional specialized admin view for `make_inline(NetworkerReplyTrigger, MessageTemplateAttachment)` so that the list_display/list_display_links/search_fields modifications are not lost. However, I'm not sure if `make_inline(NetworkerReplyTrigger, MessageTemplateAttachment)` works with `MessageTemplateAttachment` referencing a `MessageTemplate` instead of a `NetworkerReplyTrigger`. Please check to see if it does. If it doesn't work, just use a normal `class NetworkerReplyTriggerAdmin(admin.ModelAdmin)` class instead.

Reply:
> However, I'm not sure if `make_inline(NetworkerReplyTrigger, MessageTemplateAttachment)` works with `MessageTemplateAttachment` referencing a `MessageTemplate` instead of a `NetworkerReplyTrigger`.

That's what I'm doing now, and it works. Again, because `NetworkerReplyTrigger` inherits all of `MessageTemplate`'s fields, there's no reason to inline them together, as `NetworkerReplyTrigger` has all the necessary fields.